site/docs/concepts/optimization-techniques/kvcache-eviction-algorithm.md

```diff
@@ -3,3 +3,55 @@ sidebar_position: 2
 ---
 
 # KVCache Token Eviction Algorithm
+
+
+## Overview
+The cache eviction algorithm is designed to manage KV (Key-Value) cache memory for large language models (LLMs) during text generation. It determines which blocks of tokens should be evicted from the KV cache based on importance scores calculated from attention scores across different attention layers.
+
+## Conceptual Model
+The KV cache for each sequence is divided into three logical areas:
+
+![KV cache layout with cache eviction](./images/kv-cache-areas-diagram.svg)
+
+* Start Area: Initial tokens that are never evicted
+* Evictable Area: Tokens that can be evicted based on importance scores
+* Recent Area: Most recent tokens that are preserved (never evicted)
+
+The sizes of all three areas can be configured by modifying corresponding fields in a `CacheEvictionConfig` struct, which itself is a part of the pipeline-wide `SchedulerConfig`.
```

Comment:
I suggest to specify defaults. Also if there is a way to refer to the struct, let's add the reference.

Reply:
Specified the defaults since there is no linkable API reference yet.